code/src/app.css

```diff
@@ -0,0 +1,131 @@
+body {
+  background-color: black;
+  color: white;
+}
+
+
+header {
+  display:flex;
+  justify-content: center;
+  font-size: 80px;
+  margin-right: 60px;
+}
+
+a {
+  display:flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  text-decoration: none;
+  font-size: 14px;
+  font-family: Helvetica, sans-serif;
+  margin-right: 60px;
+  word-wrap: break-word;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+.album-card {
+  position: relative;
+  
+}
+.image-container {
+  position: relative;
+  display:flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 80px;
+}
+
+.image-container:hover img {
+  opacity: 0.4;
+}
+
+.image-container:hover .icons {
+  display: flex;
+}
+
+.icons-container {
+  position: absolute;
+  z-index: 1;
+  fill: white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  /* margin-right: 0; */
+  top: 45%;
+  left: 20%;
+ 
+}
+
+.icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  width: 180px;
+  justify-content: space-between;
+  margin-right: 90px;
+  display: none;
+
+}
+
+.heart,
+.dot {
+  width: 30px;
+}
+.play {
+  width: 60px;
+  transition: 0.4s ease;
+}
+.play:hover {
+  transform: scale(1.3);
+}
+
+.album-container {
+  max-width: 1200px;
```

Comment:
I would remove this and either change the media queries a little bit to better suit the widths of your cards, like this. And FYI, since you've already specified that it's `display: grid;` you don't have to specify it again.
```
@media (min-width: 650px) {
  .album-container {
    grid-template-columns: repeat(2, 1fr);
  }
}

@media (min-width: 950px) {
  .album-container {
    grid-template-columns: repeat(3, 1fr);
  }
}

@media (min-width: 1300px) {
  .album-container {
    grid-template-columns: repeat(4, 1fr);
  }
}

```

Reply:
Another approach to try out 👇 Don't change the grid, but instead, change the span of each card. For it to be fully responsive you would also need to set the `img`'s width to be no more than `100%` :)

```
.album-container {
  display: grid;
  grid-template-columns: repeat(12, 1fr);
  grid-gap: 1rem;
  position: relative;
  background-size: cover;
  min-height: 50vh;
  padding: 20px;
  margin: 0 auto;
}

.album-card {
  grid-column: span 12;
}

img {
  max-width: 100%;
}

/* Designed from Mobil first, from 668px show 2 cards */
@media (min-width: 668px) {
  .album-card {
    grid-column: span 6;
  }
}

@media (min-width: 1024px) {
  .album-card {
    grid-column: span 3;
  }
}
```

Reply:
@HIPPIEKICK I´ve experimentet with the grid and got responsiveness working correctly, yey! But now the icons are behaving weird, I have changed size and padding after the media@ but Im looking for a better solution, could you see what Im missing, I have push my changes.

Reply:
Do you mean that they are not fully centered? Your albums now have a dynamic size but your icons have not - so maybe change either the position or the size of the icons to a dynamic width? :)